core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -285,8 +285,11 @@ default String getEffectiveImage() {
 
     /**
      * If full stack traces are enabled in the resulting image
+     *
+     * @deprecated GraalVM 23.1+ will always build with full stack traces.
      */
     @WithDefault("true")
+    @Deprecated
```

Comment:
Given that we are still on 23.0, shouldn't we delay the deprecation?

Reply:
No, I think this is still fine. Users need a way to become aware. Deprecating it now is the right call, imo. On the next release using it would do nothing.